_studio/mfx_lib/encode_hw/vp9/include/mfx_vp9_encode_hw_utils.h

```diff
@@ -435,6 +435,8 @@ template <typename T> mfxStatus RemoveExtBuffer(T & par, mfxU32 id)
     public:
         MfxFrameAllocResponse();
 
+        MfxFrameAllocResponse(MfxFrameAllocResponse const &) = delete;
```

Comment:
It was disabled previously as well. You did it in more correct way according to modern c++ standard. But you didn't fix static analysis issue by this change. You need to do something with copy assignment as well (and with move ctor and move assignment)